icgcget/cli.py

```diff
@@ -112,90 +113,93 @@ def cli(ctx, config, logfile):
 @click.option('--icgc-transport-file-from')
 @click.option('--icgc-transport-memory')
 @click.option('--icgc-transport-parallel')
-def download(repo, fileid, manifest, output,
+def download(repo, fileids, manifest, output,
              cghub_access, cghub_path, cghub_transport_parallel,
              ega_access, ega_password, ega_path, ega_transport_parallel, ega_udt, ega_username,
              gdc_access, gdc_path, gdc_transport_parallel, gdc_udt,
              icgc_access, icgc_path, icgc_transport_file_from, icgc_transport_memory, icgc_transport_parallel):
     logger = logging.getLogger('__log__')
```

Comment:
By using `'__log__` or any other defined names I can get the same logger object  with the same formatting and handlers in any other module.  This also means I don't have to pass log file information around the entire project.